### DISCUSSION TITLE
Remove Combat Knife

### DISCUSSION THREAD

**Discussion by romqrocha** [zapzombies/zap-community#27] [Ideas]
**Is your feature request related to a problem? Please describe.**
Combat knife is useless because it only increase your DPS while reloading which would only be needed early game and by not that much. Late game you'll want to use Steanker 100% of the time. There's no need have a lucky chest weapon that's only there to waste 1k gold.

**Describe the solution you'd like**
remove it

**Describe alternatives you've considered**
remove

**Additional context**
remov

**Comment by Sternig** [comment]
hes telling the truth
getting combat knife 2 times in a row ruined my whole life

**Comment by ghost** [comment]
No. This is the flamethrower of Hades' Dominion.

**Comment by romqrocha** [comment]
even flamethrower is useful in bb sr

**Comment by ElliotHiester** [comment]
remove it

**Comment by tahmid-23** [comment]
this is officially a discussion

**Comment by VegetalDev** [comment]
i feel like everyone agrees it should be removed

**Reply by VegetalDev** [reply]
(except for @JolindaThique)

**Reply by tahmid-23** [reply]
as I always disagree with the popular opinion, I must say I am in support of keeping combat knife.

**Reply by ghost** [reply]
Think about it: if you're extra unlucky in this luck-baysed terrible game that Mojang themselves should maintain according to mici, you get a useless knife as punishment.

**Reply by EmperorKai2** [reply]
Why should we remove combat knife

**Reply by EmperorKai2** [reply]
It’s poggers ngl

**Comment by VegetalDev** [comment]
but let's not move active issues into discussions, i like having them all in one place

**Reply by tahmid-23** [reply]
I wanted to know what a discussion was 'cuz I've never seen this page lol

**Comment by VegetalDev** [comment]
either way, this issue should have been closed because i'm removing combat knife rn anyway

**Reply by ghost** [reply]
No.

**Reply by tahmid-23** [reply]
why not just BUFF it

**Reply by VegetalDev** [reply]
if any of you can name one reasonable way combat knife could be buffed, without making it a steanker clone, and making it at least as good as steanker, i will re-add it

**Reply by tahmid-23** [reply]
make it insta kill all enemies

**Reply by VegetalDev** [reply]
> 
> 
> make it insta kill all enemies

amazing idea 10/10

**Reply by ghost** [reply]
Make it worse so you gain a buff in pissing people off, which is good.

**Reply by EmperorKai2** [reply]
Make it paralyze mobs for a short amount of time like elder but in form of a sword, so that it’s useful on bosses (not knocking them back)

**Comment by ElliotHiester** [comment]
perfect

**Comment by Liko42069** [comment]
being removed? lets go
